File: cpp/function.cpp
#include <iostream>
void happyBirthday(std::string name, int age);
int main() {
  std::string name;
  std::cout << "Enter your name:";
  std::cin >> name;
  int age;
  std::cout << "Enter your age:";
  std::cin >> age;
  happyBirthday(name, age);
  return 0;
}
void happyBirthday(std::string name, int age) {
  std::cout << "Happy birthday to you\n";
  std::cout << "Happy birthday to you\n";
  std::cout << "Happy birthday dear " << name << '\n';
  std::cout << "Happy birthday to you\n";
  std::cout << "You are " << age << " years old\n";
}

File: rust/function.rs
fn main(){
    hello();
    tell_height(170);
    human_id("KenyuF", 17);
}
fn hello(){
    println!("Hello world");
}
fn tell_height(height: u32){
    println!("My height is {}cm", height);
}
fn human_id(name: &str, age: u32){
    println!("My name is {}, I'm {} years old", name,age);
}
